index.html

```diff
@@ -25,6 +25,20 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+
+      <nav id="mobile-menu-container">
+        <div class="menu-shape">
+          <img class="img-square" src="./media/svg/square.svg" alt="square" />
+          <i id="mobile-close-menu" class="fa fa-times"></i>
+        </div>
+        <ul>
+          <li><a class="nav-link" href="#">Hello</a></li>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li><a class="nav-link" href="#contact">Contact</a></li>
+        </ul>
+        <img class="img-circle" src="./media/svg/circle.svg" alt="square" />
```

Comment:
Same issue as mentioned above :up: 
It seems you added some extra decorations! 
Sadly this image is not present in the original template: 
# Yours
![image](https://user-images.githubusercontent.com/28903404/139599899-034424ba-5ea6-4a97-81fa-89a15fa7bca8.png)

# Original
![image](https://user-images.githubusercontent.com/28903404/139599240-b2537a67-8b50-4216-bd7d-bd194351f74b.png)

Furthermore decoration images should always be styled in the CSS as a `background-image` That way you can control its position and scale values more precisely!

But for the purpose of this project I kindly ask you to remove them so your menu follows the original template!
```suggestion
   
```